library/src/main/java/com/xfinity/blueprint/model/Component.kt

```diff
@@ -12,9 +12,17 @@
 package com.xfinity.blueprint.model
 
 import com.xfinity.blueprint.presenter.ComponentPresenter
+import com.xfinity.blueprint.presenter.DefaultComponentPresenter
 
 interface ComponentModel
 
 data class Component(val model: ComponentModel,
                      val viewType: Int,
-                     val presenter: ComponentPresenter<*, *>? = null)
\ No newline at end of file
+                     val presenter: ComponentPresenter<*, *>? = null) {
+
+    /**
+     *  Alternate constructor for creating components that are just a static view, with no presentation necessary.
+     *  Meant for things like dividers, headers, footers, etc.
+     */
+    constructor(viewType: Int) : this(object: ComponentModel {}, viewType, DefaultComponentPresenter())
```

Comment:
I like this, I was planning extensions. Is there a way to express a warning or error for components which are not fit to the description ?

Reply:
Well, you won't be able to instantiate a component unless you either have model+viewType or just viewType.  And, if you don't include a presenter, and there's no default presenter for that viewType, you'll hit a runtime exception that says there's no presenter for that view. 

Was there another type of warning were you thinking of?